app/build.gradle.kts

```diff
@@ -18,11 +20,16 @@ android {
 		vectorDrawables {
 			useSupportLibrary = true
 		}
+
+		val properties = Properties().apply {
+			load(project.rootProject.file("local.properties").reader())
```

Comment:
You can also just do `val apiKey = gradleLocalProperties(rootProject, providers).getProperty("API_KEY")` here :)

Reply:
the system didn't like the rootProject parameter and had to do some modifications to this suggestion.